config/initializers/content_security_policy.rb

```diff
@@ -1,13 +1,14 @@
 Rails.application.config.content_security_policy do |policy|
   policy.default_src  :self, :https
   policy.font_src     :self, :https, :data
-  policy.img_src      :self,
+  policy.img_src      :self, 'blob:', 'data:',
```

Comment:
Assume this is because we have to allow-list our own content

Reply:
yes